docs/release-notes/7.17.asciidoc

```diff
@@ -5,6 +5,44 @@
 [[release-notes-7.17.6]]
 === 7.17.6
 
+[discrete]
+[[known-issue-7.17.6]]
+==== Known issues
+* In some situations, {elastic-endpoint} might change to a non-running state on Windows endpoints and cannot be restarted. {agent} will appear `Unhealthy` when this happens (https://github.com/elastic/endpoint/issues/29[#29]).
++
+To determine whether {elastic-endpoint} has stopped running because of this issue, run the following PowerShell command as an administrator:
+
++
+[source,console]
+--------------------------------------------------
+PS C:\Users\user> Get-WinEvent Microsoft-Windows-CodeIntegrity/Operational | where Id -eq 3004 | where Message -match "elastic-endpoint.exe"
+
+
+   ProviderName: Microsoft-Windows-CodeIntegrity
+
+TimeCreated                      Id LevelDisplayName Message
+-----------                      -- ---------------- -------
+9/22/2022 10:47:35 AM          3004 Error            Windows is unable to verify the image integrity of the file \Device\HarddiskVolume3\Program Files\Elastic\Endpoint\elastic-endpo...
+9/19/2022 2:10:14 PM           3004 Error            Windows is unable to verify the image integrity of the file \Device\HarddiskVolume3\Program Files\Elastic\Endpoint\elastic-endpo...
+--------------------------------------------------
+
++
+
+If {elastic-endpoint} is not running, there are several workarounds you can take:
+
+** **Trigger an Endpoint reinstall by manually uninstalling {elastic-endpoint} on affected hosts**: Remove an invalid {elastic-endpoint} install by manually invoking the {elastic-endpoint} <<uninstall-endpoint,uninstall command>> on affected hosts. Once the uninstallation process has finished, use the command below to restart {agent}. Restarting {agent} automatically begins {elastic-endpoint}'s re-installation process:
```

Comment:
Kind of a silly question -- but what do we mean by `"manually invoking the {elastic-endpoint} <<uninstall-endpoint,uninstall command>> on affected hosts"`?  I understand this to mean simply mean run the uninstall endpoint command. If so, we should simplify the language -- but wanted to check to make sure I understood first. 

Alt suggestion: 
`Remove an invalid {elastic-endpoint} install by running the {elastic-endpoint} <<uninstall-endpoint,uninstall command>> on affected hosts.`

```suggestion
** **Begin an endpoint reinstall by manually uninstalling {elastic-endpoint} on affected hosts**: Remove an invalid {elastic-endpoint} install by manually invoking the {elastic-endpoint} <<uninstall-endpoint,uninstall command>> on affected hosts. Once the uninstallation process has finished, run the following command to restart {agent}, which automatically starts {elastic-endpoint}'s re-installation process:
```

Reply:
Thanks @joepeeples. One small thing -- I believe `reinstall` is sans the hyphen. Example in Elasticsearch [here](https://www.elastic.co/guide/en/elasticsearch/reference/8.4/zip-windows.html).

Reply:
Hey, @jmikell821 ! I was actually following our style guidelines for hyphenated words [here](https://docs.elastic.dev/content-design/tech-docs/writing-tips#hyphenated-and-non-hyphenated-words), but am open to following a different standard if we should be following Elasticsearch's example.